docs/content/intro/flow-token.mdx

```diff
@@ -96,7 +96,6 @@ There are a number of programs available to earn FLOW tokens.
 
 - [Alpha Program](https://www.onflow.org/flow-alpha)
```

Comment:
do we know if these programs are still being monitored?
should we add a link to the flow dev grants as well?

Reply:
not sure if they are monitored but I added the deb grants